### DISCUSSION TITLE
Map aus einer Datenbank

### DISCUSSION THREAD

**Discussion by AndreHaupt85** [neuefische/questions#171] [Web] [answered]
Hallo Leute, ich habe ein Problem das ich mir nicht erklären kann.

mit folgender Funktion ziehe ich mir die Daten aus meiner DB und logge sie.

![Bildschirmfoto 2022-09-25 um 18 01 18](https://user-images.githubusercontent.com/109967968/192153187-c2ac14ce-ea12-4e94-b25c-4ea7021ea7a2.png)

Beim Log kommen 3 verschiedene Ergebnisse raus obwohl alle Datensätze komplett gleich sind.

Einmal der Vollständige Datensatz.
![Bildschirmfoto 2022-09-25 um 18 00 47](https://user-images.githubusercontent.com/109967968/192153262-105d3fd1-35ff-4f15-8dca-b5a3e5d77dcb.png)

Einmal der Datensatz mit allen Daten aber "Type" ist undefined.
![Bildschirmfoto 2022-09-25 um 18 00 58](https://user-images.githubusercontent.com/109967968/192153293-850242ac-d243-46b4-967d-a38277d64c89.png)

Einmal mit allen Daten undefined außer der ID.
![Bildschirmfoto 2022-09-25 um 18 01 06](https://user-images.githubusercontent.com/109967968/192153312-c8087f24-ef52-43d0-aef4-4353ab5f4366.png)

Das ist der dazu gehörende Fehler im Server.
![Bildschirmfoto 2022-09-25 um 18 04 23](https://user-images.githubusercontent.com/109967968/192153339-d1f51fc6-91b8-413a-96f5-1aee2f6985ba.png)

Hat jemand eine Idee?

**Comment by DunjaWohlers** [comment]
https://stackoverflow.com/questions/36521198/how-to-return-25-results-at-a-time-with-mongoose ggf?

**Comment by NikRoe** [comment, marked as answer]
Hallo Andre,

kannst du uns noch einen Pull Request zukommen lassen?

Kannst du uns zudem einen Screenshot deiner Datenbank zuschicken?

**Reply by AndreHaupt85** [reply]
Hallo Niklas, das is der Screenshot von meiner DB. 

Einen Teil des Problems habe ich inzwischen gelöst es gibt nur noch komplett oder komplett undefined Datensätze.
das Mittelding is weg.
Pullrequest kann ich doch erst stellen wenn ich fertig bin mit der Aufgabe oder?

sonst wäre ich jetzt auch vom Arzt zurück und im Zoom.

<img width="602" alt="Bildschirmfoto 2022-09-26 um 11 50 14" src="https://user-images.githubusercontent.com/109967968/192246810-e3e3c489-ec22-43ef-aceb-b357b93fedb2.png">
